runtime/compiler/control/JitDump.cpp

```diff
@@ -169,9 +192,37 @@ jitDumpRecompileWithTracing(J9VMThread *vmThread, J9Method *ramMethod, TR::Compi
    compInfo->setVMStateOfCrashedThread(vmThread->omrVMThread->vmState);
 
    J9::JitDumpMethodDetails details(ramMethod, optionsFromOriginalCompile, isAOTBody);
-   auto rc = compilationOK;
    auto queued = false;
-   compInfo->compileMethod(vmThread, details, oldStartPC, TR_no, &rc, &queued, plan);
+   auto rc = compilationOK;
+#if defined(J9VM_OPT_JITSERVER)
+   if (compInfo->getPersistentInfo()->getRemoteCompilationMode() == JITServer::SERVER)
+      {
+      // Inform the client that the compilation thread has crashed and send it
```

Comment:
The plan we created above is not used, so may be we should free it (or not even create it)